engine/src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -43,9 +41,14 @@ class CromwellServerActor(cromwellSystem: CromwellSystem)(override implicit val
   val interface = webserviceConf.getString("interface")
   val port = webserviceConf.getInt("port")
 
-  val routeUnwrapped: Boolean = cromwellSystem.conf.as[Option[Boolean]]("api.routeUnwrapped").getOrElse(false)
-
-  val allRoutes: Route = routes.wrapped("api", routeUnwrapped) ~ swaggerUiResourceRoute
+  /**
+    * /api routes have special meaning to devops' proxy servers. NOTE: the oauth mentioned on the /api endpoints in
+    * cromwell.yaml is broken unless the swagger index.html is patched. Copy/paste the code from rawls or cromiam if
+    * actual cromwell+swagger+oauth+/api support is needed.
+    */
+  val apiRoutes: Route = pathPrefix("api")(concat(workflowRoutes))
```

Comment:
In this particular case-- taste. This is a collection all of the routes that are prefixed with "/api", in one place. Alternatively each route _could_ have define a full-path, but that wasn't my preference-- and would have required the specs to be updated to _also_ send requests to full-paths.

More than happy if one changes the route definitions and specs in a future PR.